src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const TodoForm = props => {
+    return (
+       <form>
```

Comment:
You can put your submit method on here on an `onSubmit` property to do the same thing as your button plus being able to submit using the enter key.